src/browser/runtime_p2p/peer_connection_manager.ts

```diff
@@ -1,7 +1,21 @@
 import { EmitterMap } from './emitter_map';
 import { createHash } from 'crypto';
-import { connect, Identity, Fin } from 'hadouken-js-adapter';
+import { Identity, Fin } from 'hadouken-js-adapter';
 import { EventEmitter } from 'events';
+import * as path from 'path';
+
+/*
+    Try loading local dev js-adapter, if failed, resort to
+    loading production js-adapter, which is included as a dependency
+ */
+let connect: any;
+try {
+    const jsAdapterDevPath = path.resolve(process.cwd(), 'resources', 'js-adapter', 'src', 'main.js');
+    // tslint:disable-next-line
+    connect = require(jsAdapterDevPath).connect; // DEV js-adapter
```

Comment:
Use a [dynamic import](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-4.html) rather than a require.

Reply:
I don't want to import it asynchronously